nemo/collections/cv/data/video_to_text_dataset.py

```diff
@@ -0,0 +1,361 @@
+import random
```

Comment:
There is a bunch of missing copyrights, just check all the files

Reply:
added copyrights